frontend/apps/remark42/app/components/auth/auth.messsages.ts

```diff
@@ -65,4 +65,8 @@ export const messages = defineMessages<string>({
     id: 'auth.signout',
     defaultMessage: 'Sign Out',
   },
+  noProviders: {
```

Comment:
I think you need to add a translation string and regenerate translations in case you're adding new text like that.